src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcConnectTest.java

```diff
@@ -60,17 +60,22 @@ public class ITJdbcConnectTest extends ITAbstractJdbcTest {
   @ClassRule public static JdbcIntegrationTestEnv env = new JdbcIntegrationTestEnv();
 
   private Database database;
+  private SpannerOptions options;
 
   @Before
   public void setup() {
     database = env.getOrCreateDatabase(Dialect.GOOGLE_STANDARD_SQL, Collections.emptyList());
+    options = env.getTestHelper().getOptions();
   }
 
   private String createBaseUrl() {
     StringBuilder url = new StringBuilder("jdbc:cloudspanner:");
     if (EmulatorSpannerHelper.isUsingEmulator()) {
       url.append("//").append(System.getenv("SPANNER_EMULATOR_HOST"));
     }
+    if (options.getHost() != null) {
+      url.append(options.getHost().substring(options.getHost().indexOf(58) + 1));
```

Comment:
Could we change this into something that is more logical than `58` (that is; use the `:`  char constant instead of 58)? And also add a comment for why we are taking a specific part of the string instead of the whole string.

Reply:
Thanks! Incorporated this.